doc/tracing.md

```diff
@@ -0,0 +1,34 @@
+# VCML Tracing
+
+VCML offers TLM transaction tracing functionality.
```

Comment:
VCML offers transaction tracing functionality for TLM and various other protocols, such as GPIO, CLK, PCI, I2C, SPI, SD, Serial, Ethernet, CAN and Virtio.